Web/Views/Payment/CreateInboundPayment.cshtml

```diff
@@ -51,7 +65,9 @@ else
                 bbanAccountNumber: $('#payment-target-bban-account').val(),
                 Amount: parseFloat($('#payment-amount').val()),
                 message: $('#payment-message').val(),
-                transactionText: $('#payment-transaction-text').val()
+                transactionText: $('#payment-transaction-text').val(),
+                IssuePayerToken: $('#issue-payer-token').is(':checked'),
+                PayerToken: $('#payer-token').val()
```

Comment:
I believe camelCase is common for this kind of code instead of PascalCase, so "IssuePayerToken" -> "issuePayerToken" and "PayerToken" -> "payerToken"